test/WebJobs.Script.Tests/FunctionMetadataManagerTests.cs

```diff
@@ -438,6 +438,43 @@ public void FunctionMetadataManager_ResetProviders_OnRefresh()
             Assert.Equal("newFunction", testFunctionMetadataManager.GetFunctionMetadata(true).FirstOrDefault()?.Name);
         }
 
+        [Fact]
+        public void FunctionMetadataManager_GetsMetadata_ValidateHostJson()
+        {
```

Comment:
Is it possible to add test cases for other scenarios also - where Host.json is present at root or in a subdirectory or at both the places? If not, we can test these scenarios using private site extension.